pod.go

```diff
@@ -584,8 +595,8 @@ func (p *Pod) delete() error {
 		return err
 	}
 
-	if state.State != StateReady && state.State != StateStopped {
-		return fmt.Errorf("Pod not ready or stopped, impossible to delete")
+	if state.State != StateReady && state.State != StatePaused && state.State != StateStopped {
```

Comment:
Is that okay to consider a pod can be deleted if it is paused ? @sameo any thoughts on that ?